### DISCUSSION TITLE
SSH identity_file

### DISCUSSION THREAD

**Discussion by jaylinnyc** [martomi/chiadog#201] [Q&A] [answered]
Hi, I'm followed the instructions to setup monitoring harvester remotely.    When I connect to the harvester server using "ssh -i ~/.ssh/key"   it works fine.   My question is, where do I specify the identity_file in the config.yaml?

I'm getting the following error:

"paramiko.ssh_exception.SSHException: No authentication methods available"

This is due to lack of identity_file.

**Comment by pieterhelsen** [comment, marked as answer]
Hi,

Please follow the second part of this guide labelled 'Adding your SSH key to the SSH-agent':
https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

**Reply by jaylinnyc** [reply]
Thank you!  It's exactly what I was missing.
